uzplayer/src/main/java/com/uiza/sdk/view/UZVideoView.java

```diff
@@ -1296,8 +1301,10 @@ public RelativeLayout getRlLiveInfo() {
         return rlLiveInfo;
     }
 
-    public TextView getTvLiveView() {
-        return tvLiveView;
+    public void setLiveCCU(int ccu) {
```

Comment:
Please change `setLiveCCU` to `setLiveViewers` to match with our current term. And we dont use the term `ccu`